cli/cli_connect_test.go

```diff
@@ -18,39 +20,92 @@ import (
 	"google.golang.org/grpc"
 )
 
+func captureOutput(f func()) (string, error) {
+	reader, writer, err := os.Pipe()
+	if err != nil {
+		return "", err
+	}
+	stdout := os.Stdout
+	stderr := os.Stderr
+	defer func() {
+		os.Stdout = stdout
+		os.Stderr = stderr
+	}()
+	os.Stdout = writer
+	os.Stderr = writer
+	out := make(chan string)
+	wg := new(sync.WaitGroup)
+	wg.Add(1)
+	go func() {
+		var buf bytes.Buffer
+		wg.Done()
```

Comment:
`wg` seems useless here and also it is unnecessary because syncing is done via channel

Reply:
true. I'll refactor this and remove completely the coroutine, because it just ads unneeded complexity.